packages/ui/src/components/molecules/token-icon/token-icon.tsx

```diff
@@ -0,0 +1,84 @@
+import { DEFAULT_ICON_SIZE } from "../../../icons/constants";
+import {
+  MilkTiaIcon,
+  StrideIcon,
+  CelestiaIcon,
+  StrideTiaIcon,
+  UsdcIcon,
+  WrappedTiaIcon,
+  DotIcon,
+} from "../../../icons";
+
+export const TokenIcon = ({
+  symbol,
+  size = DEFAULT_ICON_SIZE,
+  className = "",
+}: {
+  symbol: string | null;
+  size?: number;
+  className?: string;
+}) => {
+  const tokenKey = symbol?.toLowerCase() || "unknown";
```

Comment:
I don't think this magic string ("unknown") is the best way to implement this. I'll send you what I'm thinking in dm.

Reply:
Discussed over DM